pkg/kubelet/rkt/rkt.go

```diff
@@ -1154,20 +1230,18 @@ func (r *Runtime) waitPreStopHooks(pod *api.Pod, runningPod *kubecontainer.Pod)
 
 	errCh := make(chan error, 1)
 	go func() {
-		if err := r.runPreStopHook(pod, runningPod); err != nil {
+		if err := r.runLifecycleHooks(pod, runningPod, lifecyclePreStopHook); err != nil {
 			errCh <- err
```

Comment:
Should the log statement be here instead so it's not lost if it errors out after the grace period timeout?